tests/test_flow.py

```diff
@@ -2721,7 +2722,9 @@ def test_write_to_parquet_partition_by_hash(tmpdir):
     read_back_df = pd.read_parquet(out_file, columns=columns)
     read_back_df.sort_values("my_int", inplace=True)
     read_back_df.reset_index(drop=True, inplace=True)
-    assert read_back_df.equals(expected), f"{read_back_df}\n!=\n{expected}"
+    # with the introduction of s, ms, us time resolutions in pandas-2.0, the dtype of the parquet data
```

Comment:
I'm pretty sure the time resolution exists in pandas 1 too.

Reply:
https://levelup.gitconnected.com/welcoming-pandas-2-0-194094e4275b

Reply:
Turns out it was supported in numpy `datetime` but not pandas `Timestamp`.

Reply:
https://numpy.org/doc/stable/reference/arrays.datetime.html